### PR TITLE
Rename About page to "Contact Us", add Discord and GitHub links

### LINKED ISSUE #436: Add a contact us page
We need to add a "Contact us" page where we list our contact email, our Discord, and GitHub organization so that our users can find us.

Maybe we can repurpose the About page for this, which reminds me: I'm not sure `hello@hollowverse.com` email even works! We need to fix that. We can create a G Suite Group for that email.

### PR DESCRIPTION
**Pull request by forabi**
Closes #436

### CODE REVIEW

**Comment by forabi** [comment]
@wholesomedev Are you okay with the copywrite?

I've already created the group hello@hollowverse.com and allowed public posts.

File: src/app/pages/ContactUs/ContactUs.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  'https://discordapp.com/channels/308394001789353985/308394051114237952';
+
+export const ContactUs = () => (
+  <TextPage>
+    <h1>Contact Us</h1>
+    <p>
+      It is often said that one should avoid discussing politics or religion in
+      polite company. But these are some of the most interesting topics and I
+      hope I can get you talking about them!
+    </p>
```
Comment (wholesomedev) [inline src/app/pages/ContactUs/ContactUs.tsx]: Can we use this copy:

```
It is often said that one should avoid discussing politics or religion in
polite company. But these are some of the most interesting topics and we
hope to get you talking about them!
```

**Review by wholesomedev — APPROVED** [review]
This looks good. Just one comments. And we should change the "About" anchor text in the footer to "Contact us"

**Review by wholesomedev — COMMENTED** [review]
LGTM

**Pull request merged**